canopen/objectdictionary/__init__.py

```diff
@@ -13,6 +13,20 @@
 logger = logging.getLogger(__name__)
 
 
+def export_od(od, dest=None, docType="eds"):
```

Comment:
Maybe the doc type could be inferred from the file extension if left out?

Reply:
Yeah that makes sense. Though i originally thought that dest should only be a file descriptor. Now i will also accept a filename.